### DISCUSSION TITLE
Add account creation date property to app.bsky.actor

### DISCUSSION THREAD

**Discussion by itsWindows11** [bluesky-social/atproto#1581] [Bluesky Lexicons]
**Is your feature request related to a problem? Please describe.**

Currently getting the account creation date depends on heuristics. It would be good to have a straightforward method to get the account creation date in the app.bsky.actor lexicon.

**Describe the solution you'd like**

Add a property `accountCreationDate` to either the basic or detailed profile view.

**Describe alternatives you've considered**

The heuristics I'm talking about, which is basically just getting the first action on a repo but this can be inaccurate.

**Additional context**

*No response*

**Comment by kjarex** [comment]
I'm not proposing against your request. Just as an alternative solution or workaround until something better might show up:

It's not part of the lexicon, but at least it's accurate and easily to get:
Query the appropriate plc.directory for the did's log/audit. Among other things it contains the creation timestamp (in <first object>.cid.createdAt)

**Reply by bnewbold** [reply]
This is one way to get this info, though note that it only works for `did:plc` DIDs, not `did:web`.

**Reply by kjarex** [reply]
Thank you, I really appreciate the note, as I could have been unaware and therefore could have unknowingly run into issues. But I pretend did:web doesn't exist (well, right now it de facto does not, but unfortunately it will eventually), as it shouldn't as it's an abomination which repeats just one of the biggest flaws of ActivityPub. But thanks 🙂

**Reply by DavidBuchanan314** [reply]
@kjarex did:web does exist already, I have a live account that uses it: https://bsky.app/profile/did:web:retr0.id

**Reply by kjarex** [reply]
I’m still going to pretend it doesn’t and won’t ever going to support it ☺️

**Reply by bnewbold** [reply]
that is too bad, it is really pretty easy to support `did:web`
